playbooks/deploy_vms_cluster.yaml

```diff
@@ -21,8 +21,9 @@
         command: create
         system_image: /tmp/os_{{ item }}.qcow2
         force: true
-        live_migration: true
+        live_migration: "{{ true if livemigration_user is defined }}"
```

Comment:
shouldn't this condition be on the line just after this one?

Reply:
No.
`live_migration` define if we want to use livemigration or not.
`migration_user` define the user to do it.
In the case of our ansible inventory, live migration is active only if we define a user for it. So only if the variable `livemigration_user` is defined.

But I forget to explain that in the ansible inventory. I do it

Reply:
ok but maybe this condition should also be on the migration_user line below.
If livemigration_user is not defined, won't the playbook fail otherwise?